Relative imports break go get

```
$ go get -v github.com/slyrz/mango
github.com/slyrz/mango (download)
/Users/coda/Projects/go/src/github.com/slyrz/mango/mango.go:13:2: local import "./markup" in non-local package
```

If you use `"github.com/slyrz/mango/markup"` and `"github.com/slyrz/mango/source"` instead, Mango will play nicely with the rest of the Go build tools.

(Very cool project, BTW. Looking forward to using it.)

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ the comments and *flag* function calls found in your .go files.
 
 Execute
 ```bash
-go build mango.go
+go install github.com/slyrz/mango
 ```
 to build mango.
 

--- a/mango.go
+++ b/mango.go
@@ -10,13 +10,14 @@
 package main
 
 import (
-	"./markup"
-	"./source"
 	"flag"
 	"fmt"
 	"os"
 	"path"
 	"strings"
+
+	"github.com/slyrz/mango/markup"
+	"github.com/slyrz/mango/source"
 )
 
 var (

